core/services/dependencies/ClassAlias.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace EventEspresso\core\services\dependencies;
+
+use EventEspresso\core\exceptions\InvalidAliasException;
+
+/**
+ * Class ClassAlias
+ * Simple value object for representing a class alias such as an interface
+ *
+ * @package EventEspresso\core\services\dependencies
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class ClassAlias
```

Comment:
Or maybe `ConcreteClassOf`?

Reply:
No because the intent is to be an alias, which just means something is using another name.
Whether that means an interface or child class doesn't matter, we are just using this VO as a way to indicate that a class name is being swapped out for another.